### ISSUE TITLE
Archlinux Packaging Out of Date

### ISSUE THREAD

**Issue by kousu** [fangq/jsonlab#64]
[This note](https://github.com/fangq/jsonlab#install-jsonlab-on-arch-linux)

> sudo pacman -S jsonlab

doesn't work:

```
$ sudo pacman -Sy jsonlab
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 multilib is up to date
error: target not found: jsonlab
```

It got moved to the AUR I guess: https://aur.archlinux.org/packages/jsonlab

so you need to `pikaur -S jsonlab` or `yaourt -S jsonlab`.

**Comment by fangq** [comment]
@kousu, I am making a new release of jsonlab (v2.0). Do know how to submit a new package for Arch? I am happy to create a compatible package or contact the maintainer for help.

**Comment by kousu** [comment]
@fangq that's great to hear. In order to get packages into Arch they first need to go through the [AUR](https://aur.archlinux.org/); if they are popular enough they will [be adopted into Arch officially](https://wiki.archlinux.org/index.php/Official_repositories#community), but it's not unusual for packages to live happily in the AUR and get lots of use for years.

Since jsonlab is already in the AUR, I think once you make an official release here on Github you could click "Flag package out of date" on https://aur.archlinux.org/packages/jsonlab; then billiepilgrim should get notified and be able to update it to use your latest version. You could also make your own AUR package -- based on https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=jsonlab -- and submit it under a different name.


Was `jsonlab` really included in Arch before? Did it get demoted from the core distribution to the AUR?

In either case, in this repo, I think you need to update your install instructions to say

> Install [jsonlab](https://aur.archlinux.org/packages/jsonlab) from the [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages)

or

> Install `jsonlab` from the AUR with
> ```
> pikaur -Sy jsonlab
> ```

(which uses the nice `pikaur` pacman+AUR wrapper, but requires the user to have installed it manually first from https://aur.archlinux.org/packages/pikaur/)

**Comment by kousu** [comment]
Another thing you could do is make a `jsonlab-git` AUR package: one that is always built from your latest master branch.  The trick is just to set `source=("git+https://https://github.com/fangq/jsonlab.git")`; for example, here's how `abiword-git` works: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=abiword-git

**Comment by fangq** [comment]
@kousu, I just released jsonlab 2.0, and flagged the arch package "out-of-date".
hopefully someone will help update the package. unfortunately I don't have any packaging experience with aur.
